### PR TITLE
add option to skip TLS verification of server's cert

### PR DESCRIPTION
**Pull request by andreykaipov**
Closes https://github.com/terraform-providers/terraform-provider-powerdns/issues/21. Lemme know if you guys prefer something else.

### CODE REVIEW

File: powerdns/provider.go [hunk 1]
```diff
@@ -20,6 +20,12 @@ func Provider() terraform.ResourceProvider {
 				DefaultFunc: schema.EnvDefaultFunc("PDNS_SERVER_URL", nil),
 				Description: "Location of PowerDNS server",
 			},
+			"skip_tls_verify": {
+				Type:        schema.TypeBool,
+				Required:    true,
```
[the remark this answers is not in the document]
Comment (andreykaipov) [reply powerdns/provider.go]: Oh yup you're right! Good catch. I meant for this to be optional like in the docs. Thank you for updating it :-)

**Comment by andreykaipov** [comment]
Fixed the merge conflict. Rebased off of master.

**Comment by splashx** [comment]
@andreykaipov I've also added support for custom CA cert file (to support a more secure config in cases one doesn't want to skip tls verify). If you're okay with this I think we can merge this PR.

**Comment by andreykaipov** [comment]
@splashx Of course! 😄 Thank you for improving it!

**Comment by splashx** [comment]
After some thought I've renamed the options to:
- `insecure_https` (instead of `skip_tls_verify`): Reasoning [1](https://github.com/terraform-providers/terraform-provider-consul/blob/master/consul/config.go#L22) + it's better to be more generic (yesterday it was SSL, today is TLS, but _it has always been_ https). Plus, from a security stand point, it's always better to explciitly say this options is nonethless _insecure_.
- `ca_certificate` (instead of `ca_certfile`): Reasoning - because [it can accept a file _or_ a content](https://github.com/hashicorp/terraform/blob/master/helper/pathorcontents/read.go#L11-L16), the var name can stay ambiguous. The documentation however describes the behavior.

**Pull request merged**